synonymdev/bitkit-react-native
# feat(recovery): switch to quick-action to enter recovery

### Description

Switches to a more reliable way of entering recovery by adding a quick-action for it.

### Type of change


### Tests


### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8538369/ac06ed74-b12f-4b93-9300-80c6b2f57a8a

### QA Notes

The quick-action will be added only after starting the app once, should be fine since there is no reason to enter recovery on a fresh install.
